### DISCUSSION TITLE
Weird auto-pair behaviour in TypeScript

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#15686] [Feature Requests]
When I import an item inside brackets, it adds an additional closing bracket. Why does this happen? Is there a way I can fix it?

[Screencast_20240802_212431.webm](https://github.com/user-attachments/assets/29102345-4595-417d-9828-54c645449233)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
